Sources/MapboxNavigation/NavigationViewLayout.swift

```diff
@@ -374,4 +374,25 @@ extension NavigationView {
         
         NSLayoutConstraint.activate(bottomBannerContainerViewLayoutConstraints)
     }
+    
+    func setupEndOfRouteConstraints() {
+        guard let endOfRouteView = endOfRouteView,
+              let endOfRouteShowConstraint = endOfRouteShowConstraint,
+              let endOfRouteHeightConstraint = endOfRouteHeightConstraint else { return }
+        
+        NSLayoutConstraint.deactivate(endOfRouteViewLayoutConstraints)
+                                
+        let endOfRouteLeadingConstraint  = endOfRouteView.leadingAnchor.constraint(equalTo: bottomBannerContainerView.leadingAnchor)
+        
+        let endOfRouteTrailingConstraint  = endOfRouteView.trailingAnchor.constraint(equalTo: bottomBannerContainerView.trailingAnchor)
```

Comment:
```suggestion
        let endOfRouteLeadingConstraint = endOfRouteView.leadingAnchor.constraint(equalTo: bottomBannerContainerView.leadingAnchor)
        
        let endOfRouteTrailingConstraint = endOfRouteView.trailingAnchor.constraint(equalTo: bottomBannerContainerView.trailingAnchor)
```

Reply:
Text and views in end of route view are not centered correctly now:

![Simulator Screen Shot - iPhone 14 Pro Max - 2022-11-15 at 13 10 46](https://user-images.githubusercontent.com/1496498/202027599-ac326c6d-a62f-49df-ba1d-0fb772935f01.png)

I think that we should not constraint to leading safe area insets there.

Reply:
I think we should still keep it constrained to the `BottomBanner`. The incorrectly centered labels were because they were constrained to the `EndOfRouteContentView` whose safe area was far from the `BottomBanner`'s (and the edge of the screen). Fixed now.